jsmvc/index.html

```diff
@@ -387,7 +387,7 @@ <h1 class="upper">JavaScript MVC</h1>
 			</div>			
 		</div>
 
-		<script src="../common/js/head.min.js"></script>
+		<script src="../common/lib/js/head.min.js"></script>
```

Comment:
Went with what the existing reveal folder structure was:
https://github.com/hakimel/reveal.js/#folder-structure

We have other 3rd party js libraries that live in top levele `js/` (e.g. Bootstrap) but the `lib/js` refers to 3rd party dependencies `reveal` has?